### DISCUSSION TITLE
Signal Type Error connecting to RavenDB in AWS Lambda

### DISCUSSION THREAD

**Discussion by nickchampion** [ravendb/ravendb#14941] [General]
Hi, I have deployed some code to a Lambda function on AWS and cannot get a connection to the database, I'm getting the following error

`RavenException: An exception occurred while contacting https://server-node/databases/db/queries?queryHash=9e848c7b0a74c499b2cfbe5a5ae6d864&addTimeSeriesNames=true . \nTypeError: Expected signal to be an instanceof AbortSignal\n    at new lh (/var/task/index.js:42:419038)\n    at /var/task/index.js:42:421446\n    at new Promise (<anonymous>)\n    at Tv (/var/task/index.js:42:421411)\n    at FE.<anonymous> (/var/task/index.js:49:5826)\n    at Generator.next (<anonymous>)\n    at /var/task/index.js:49:3670\n    at new Promise (<anonymous>)\n    at R9 (/var/task/index.js:49:3467)\n    at FE.send (/var/task/index.js:49:5544)\n    at Un.<anonymous> (/var/task/index.js:227:58781)\n    at Generator.next (<anonymous>)\n    at /var/task/index.js:227:43884\n    at new Promise (<anonymous>)\n    at ci (/var/task/index.js:227:43681)\n    at Un._send (/var/task/index.js:227:58645)\n    at Un.<anonymous> (/var/task/index.js:227:57631)\n    at Generator.next (<anonymous>)\n    at /var/task/index.js:227:43884\n    at new Promise (<anonymous>)\n    at ci (/var/task/index.js:227:43681)\n    at Un._sendRequestToServer (/var/task/index.js:227:57471)\n    at Un.<anonymous> (/var/task/index.js:227:56155)\n    at Generator.next (<anonymous>)\n    at /var/task/index.js:227:43884\n    at new Promise (<anonymous>)\n    at ci (/var/task/index.js:227:43681)\n    at Un._executeOnSpecificNode (/var/task/index.js:227:55275)\n    at Un.<anonymous> (/var/task/index.js:227:66135)\n    at Generator.next (<anonymous>)\n    at /var/task/index.js:227:43884\n    at new Promise (<anonymous>)\n    at ci (/var/task/index.js:227:43681)\n    at Un._handleServerDown (/var/task/index.js:227:65425)\n    at Un.<anonymous> (/var/task/index.js:227:58530)\n    at Generator.throw (<anonymous>)\n    at u (/var/task/index.js:227:43788)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\nThe server at https://server-node/databases/db/queries?queryHash=9e848c7b0a74c499b2cfbe5a5ae6d864&addTimeSeriesNames=true responded with status code: 503: Expected signal to be an instanceof AbortSignal`

Any ideas as to what might be causing this.

I can run the same code successfully outside of the lambda

Thanks

**Comment by ayende** [comment]
This is probably this issue:

https://stackoverflow.com/questions/59737987/nodejs-abort-fetch-in-lambda-aws-environment
